viv-wallet-app/src/views/Wallet.vue

```diff
@@ -41,27 +42,26 @@ export default class wallet extends Vue {
     errored = false;
     walletService = new WalletService();
     loginService = new LoginService();
+    usersService = new UsersService();
 
     balance = 0;
-    userId = "1";
-    userFullName = "Théophile Montgommery";
+    userId = "";
+    user = {};
 
     async mounted() {
         try {
-            [
-                this.balance,
-                this.actions,
-                this.payments,
-                this.userRoles,
-                this.userId,
-                this.userFullName
-            ] = await Promise.all([
+            if (this.$route.params.consultantId) {
+                this.userId = this.$route.params.consultantId;
+            } else {
+                this.userId = (await this.loginService.getUserId()) || "1"; // to avoid crash (no authorization service yet)
```

Comment:
it's better to add a TODO
it's easier to find later on a grep...

Reply:
Of course, I will change that.